R/prerun.R

```diff
@@ -109,3 +109,91 @@ hash_info_inputs <- function(inputs) {
   }, USE.NAMES = FALSE)
   digest::digest(inputs$md5sum[order(inputs$role)])
 }
+
+prerun <- function(queue) {
+  function(input) {
+    files <- jsonlite::fromJSON(input, simplifyVector = FALSE)
+    all_files <- c(files$inputs, files$outputs)
+    lapply(c(names(all_files)), function(name) {
+      file <- all_files[[name]]
+      if (!file.exists(file$path)) {
```

Comment:
I suspect you'll find it nicer to work with a vectorised version of this that tells you about all missing files at once.

so,


```
msg <- all_files[!file.exists(all_files)]
if (length(msg) > 0) {
  hintr_error(....build a nice error message here...)
}
```

Reply:
Tidied this up now